lib/web_ui/README.md

```diff
@@ -99,7 +116,7 @@ this option is particularly effective for building on low-powered laptops.
 ### Test browsers
 
 Chromium, Firefox, and Safari for iOS are version-locked using the
-[browser_lock.yaml][2] configuration file. Safari for macOS is supplied by the
+[browser_lock.yaml][3] configuration file. Safari for macOS is supplied by the
 computer's operating system. Tests can be run in Edge locally, but Edge is not
```

Comment:
Is there a way to determine the version of safari that's running in CI? I've tried myself and the journey is pretty cryptic :P

Reply:
I honestly don't know.